tests/test_cli.py

```diff
@@ -255,3 +259,23 @@ def test_python_m_execution():
     )
     assert result.returncode == 0
     assert "Usage: python -m cyfi" in result.stdout
+
+
+def test_cyfi_explorer_launches(tmp_path):
+    shutil.copy(ASSETS_DIR / "experiment" / "preds.csv", tmp_path / "preds.csv")
+    shutil.copy(
+        ASSETS_DIR / "experiment" / "sentinel_metadata_test.csv",
+        tmp_path / "sentinel_metadata.csv",
+    )
+
+    proc = subprocess.Popen(
+        ["cyfi", "visualize", str(tmp_path)],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        text=True,
+    )
+    time.sleep(10)
+    proc.send_signal(signal.SIGINT)
+    stdout, stderr = proc.communicate()
```

Comment:
Set it make sense to check that the exit code is `0`?

Reply:
you mean instead of killing the process? i think we'd just be checking that the sigint worked as expected, not anything about the cyfi explorer

Reply:
I guess it may be unnecessary depending on how the Gradio application works, but it's a way to test that it doesn't have a runtime error _after_ the "Running on" string that you are testing for. It might be that it doesn't make sense for this case.